docker-compose.yml

```diff
@@ -31,4 +31,4 @@ services:
     volumes:
       - ./:/code
     ports:
-      - "8001:8001"
+      - "8010:8010"
```

Comment:
Why do you change the port ? I don't understand this change.
And if it's correct to change it, we should also update the documentation

https://github.com/Volontaria/API-Volontaria/blob/9bd7c6352cb93f58e309a8eaa601b687f215e504/docs/getting_started/docker.md?plain=1#L49-L58